doc/SpecifyingApplications/CreatingSequenceDiagrams/CreatingSequenceDiagrams.md

```diff
@@ -0,0 +1,160 @@
+
+Check how the already existing diagrams of other applications look like to get an idea for how your application's diagrams need to be created!
+
+The sequence diagram consists of several building blocks.   
+The picture shows an example from the NotificationProxy:  
+![Diagram_building_blocks](./pictures/createSeqDiagram_01.png) 
+
+**1: Diagram name = *.pgn* file name**  
+* right after the opening `@startuml` the diagram name has to be provided
+* this is the same string as in the filename without the *.plantuml* file ending
+* when the diagram is exported to *.png*, the created picture file will be named with the string provided here
+
+**2: Diagram Title = ForwardingName**
+* the title is what will be shown as title in the rendered diagram
+* here, the name of the forwarding has to be provided
+* to come up with a proper name, it is advised to read [Structure of Internal _ForwardingNames_](../../ElementsApplicationPattern/Names/StructureOfInternalForwardingNames/StructureOfInternalForwardingNames.md)
+
+**3. Participants = diagram nodes**
+* the participant block is used for easier editing of the diagram
+* for each node in the diagram create a participant
+  * format: `participant "<text to be displayed in the diagram>" as <alias>`
+  * the `"<text>"` part contains what is shown in the rendered diagram, the `<alias>` is the reference to the node in the *.plantuml* raw view
+* the participant ordering determines the order in which the nodes are displayed in the diagram
+* typically, the first participant is the iniator of the request
+  * some examples for the `"<text>"` are "Admin", "/v1/embed-yourself", "external", "cyclic", a specific other application's (abbreviated) name or a combination of those
+  * chose the alias according to the node type, this can e.g. be a requestor or subscriber
+* if a node refers to a certain service it should be provided in the following format: `<AppAbbreviation>://<operation-name>`, e.g. *"NP://v1/listen-to-controller"* or *"MWDI://v1/provide-list-of-connected-devices"*
+  * including the AppAbbreviation allows to directly see to which application the operation server or operation client belongs to
+  * the service from the ServiceList for which the diagram was created, typically is the operation server, whereas other operation-names included in the diagram called due to that operation server, are the operation clients (either from the same app or a 3rd app)
+
+**4. Activation of the Forwarding**
+* add a connection between the operation server and it's preceding participant
+* then activate the operation server to indicate over which process steps the forwarding is active
+* once the forwarding is finished, deactivate the operation server participant
+* in the rendered diagram the activation is shown as a white vertical rectangle 
+
+**5. Notes Boxes**
+* sometimes additional information needs to be provided, e.g. if there are if-then-else conditions, which cannot be properly modelled in PlantUML

```

Comment:
Could you please add after "if there are if-then-else conditions ":
"or repetitions (for all ...),"